src/include/sof/math/log.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_TWO_LOGARITHM_H__
+#define __SOF_MATH_BASE_TWO_LOGARITHM_H__
+
+/* Include Files */
+#include <stdint.h>
+
+/* Defines Constant*/
+#define BASE2LOG_WRAP_SCHAR_BITS 0xFF
```

Comment:
is this expected to be used outside the function? if not is should be moved to the c file

Reply:
I prefer constant defines in a header file rather in an opposite way.